src/cellars/cellar_v2.rs

```diff
@@ -122,7 +130,7 @@ pub fn get_encoded_governance_call(
     proposal_id: u64,
 ) -> Result<Vec<u8>, Error> {
     match function {
-        InitiateShutdown(_) => {
+        GovernanceFunction::InitiateShutdown(_) => {
```

Comment:
For the ones we are moving to the SP side as per the above comments, we could either leave copies here so they are callable or remove them.